LogTarget.php

```diff
@@ -165,6 +165,21 @@ protected function collectSummary()
     }
 
     /**
+     * Returns request time in seconds
+     *
+     * @return float request time in seconds.
+     * @see http://php.net/manual/en/function.microtime.php#example-2550
+     * @since 2.0.8
+     */
+    protected function getRequestTime()
```

Comment:
Sorry this is my typo: in case `$_SERVER['REQUEST_TIME_FLOAT']` IS NOT AVAILABLE - will produce different results as it calls `time()`.